### ISSUE TITLE
Nearest neighbor border line

### ISSUE THREAD

**Issue by matthiasbruch** [GoodBoyDigital/pixi.js#1499]
Hi,

Thanks for your great work!

I'm currently trying to implement zooming in with pixi, but I got a last issue: You always recommend to use nearest neighbor mode for scaling to get crispier lines (PIXI.scaleModes.DEFAULT = PIXI.scaleModes.NEAREST) ... unfortunately there's still that "last border" which mixes up the two nearest colors. Is it possible to get rid of that line as well? I really just want either the one color or the other...

After switching from linear to nearest I ALMOST got what I needed, but I would have to implement a workaround for these "border areas" and wanted to ask once before.

**Comment by englercj** [comment]
Do you have a running example? I literally have no idea what you are talking about when you say "border areas".

**Comment by matthiasbruch** [comment]
1.) Thanks for your very very fast reaction.
2.) Forget what I wrote - I tried to rebuild the example and it worked - first I thought it would be the new release, but it was actually just the damn browser zoom that automatically occures, when I don't use the monitor with my laptop ... The browse set it's zoom to 110% ... without that zoom everything is really really great and crisp - thanks for your cool library!

If you still want to see, what I saw - render two colored blocks that overlap, set zoom to 110% and you'll get a very thin line where the browser mixes the colors on the overlapping. (sorry for false-alarm :-D)
